Makefile

```diff
@@ -1,13 +1,14 @@
-NOCURL := 0
-STATIC := 1
+NOCURL := 1
+STATIC := 0
 
-CURL   := E:\Dev\eclipse-workspace\curl
+
+CURL := /opt/homebrew/opt/curl

```

Comment:
Why not add Makefile.macos instead of this? This does not make this "macos compatible" it makes it "works only on macos"